app/models/planet.py

```diff
@@ -0,0 +1,21 @@
+class Planet:
+    def __init__(self, id, name, description, signs_of_life=None):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.signs_of_life = signs_of_life
+    
+    def get_dict(self):
+        return {
+            "id" : self.id,
+            "name" : self.name,
+            "description" : self.description,
+            "signs of life" : self.signs_of_life
+        }
+
+planets = [
```

Comment:
🤩 Love the exoplanets! Though probably going to have to disagree about life on WASP-107 b. 😉